lib/testing/src/lib/core/pages/login-sso.page.ts

```diff
@@ -53,6 +72,31 @@ export class LoginSSOPage {
         await browser.waitForAngular();
     }
 
+    async loginBasicAuth(username, password): Promise<void> {
+        await this.goToLoginPage();
+
+        await LocalStorageUtil.clearStorage();
+        await LocalStorageUtil.setStorageItem('providers', browser.params.testConfig.appConfig.provider);
```

Comment:
could the provider be read from the .env file instead of changing it in the localStorage?